src/validation/fields/helper/github-handles.validation.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { ErrorMessages } from '../../error.messages';
+
+export const validateGithubHandles = (value: string) => {
+    if (!value) {
+        throw new Error(ErrorMessages.GIT_HANDLE);
+    }
+
+    const handles = value.split(',').map(handle => handle.trim());
+    const githubHandleRegex = /^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$/;
+
+    for (const handle of handles) {
+        if (!githubHandleRegex.test(handle) || handle.length > 39 || handle.length < 1) {
```

Comment:
Possible improvement to this is splitting out the `if` checks and having a custom error message for each one (like on the GitHub UI). It's hard to tell where the actual error is (say you submitted 10 handles and one errors, how would you know which one errored?). Will require a bit of thinking, what do you think?